src/main/java/com/wojciechbarwinski/demo/legendary_warehouse/dtos/InsufficientStockDTO.java

```diff
@@ -0,0 +1,4 @@
+package com.wojciechbarwinski.demo.legendary_warehouse.dtos;
+
+public record InsufficientStockDTO(String productId, int expectedQuantity, int stockQuantity) {
```

Comment:
Ze skrótami czy akronimami na dwoje babka wróżyła - jedni wolą XxxYyyDTO, inni XxxYyyDto jakby co ;) to samo było z RESTController (aczkolwiek tutaj raczej spotyka się RestController)

Reply:
oki, rozumiem
mi jakoś łatwiej z DTO, łatwiej rzuca sie w oczy i od razu wiesz co to. Ale nie problem zmienić